hasher-matcher-actioner/hmalib/common/evaluator_models.py

```diff
@@ -35,5 +25,5 @@ class ActionRule(HMAConfig):
     """
 
     action_label: ActionLabel
-    must_have_labels: t.Set[Label]
-    must_not_have_labels: t.Set[Label]
+    must_have_labels: t.Set[ClassificationLabel]
+    must_not_have_labels: t.Set[ClassificationLabel]
```

Comment:
Was there a reason you changed the typing of these two to ClassificationLabel? It's not that it's a bad thing, but you haven't then made other compensating changes elsewhere. For example, line 142 in action_evaluator.py is a function that takes in classifications as just labels to be compared with rules' must have and must not have labels. Just wondering what problem you were trying to solve, and if it is necessary, maybe do another check for instances like line 142 in action_evaluator.py.